app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesPresenter.java

```diff
@@ -112,7 +112,6 @@ private void listenToModelChanges() {
 
         attendeeListener.getNotifier()
             .filter(attendeeModelChange -> attendeeModelChange.getAction().equals(BaseModel.Action.UPDATE))
-            .distinctUntilChanged()
```

Comment:
the question is how `distinctUntilChanged` is going to compare model? Don't we have to set that logic?